nucliadb/src/nucliadb/search/api/v1/find.py

```diff
@@ -126,6 +126,16 @@ async def find_knowledgebox(
     security_groups: list[str] = fastapi_query(SearchParamDefaults.security_groups),
     show_hidden: bool = fastapi_query(SearchParamDefaults.show_hidden),
     reranker: Reranker = fastapi_query(SearchParamDefaults.reranker),
+    user_metadata: Optional[str] = Query(
```

Comment:
I would rename this to be more specific , so it is clear that is just for audit
```suggestion
    audit_metadata: Optional[str] = Query(
```
wdyt?

Reply:
Yeah, it looks better, thanks